### ISSUE TITLE
Add support for tesseract-ocr

### ISSUE THREAD

**Issue by MRayermannMSFT** [Yihcheng/Project2020#12]
Tesseract is the current goto for OCR libraries. It's fast and runs locally. See here https://github.com/tesseract-ocr/tesseract

**Comment by yihcheng** [comment]
Does it support non-English characters, like Chinese, Japanese and so on?

**Comment by MRayermannMSFT** [comment]
Yes, though if you are looking for non-English/many languages, you have to specify at runtime.

https://github.com/tesseract-ocr/tesseract/wiki#other-languages
